dsgrid/config/dataset_schema_handler_one_table.py

```diff
@@ -79,3 +83,12 @@ def _check_one_table_data_consistency(self, config: DatasetConfig, load_data):
                 raise DSGInvalidDataset(
                     f"load_data records do not match dimension records for {name}"
                 )
+
+    def get_unique_dimension_rows(self, mapping_references):
+        time_dim = self._config.get_dimension(DimensionType.TIME)
+        time_cols = set(time_dim.get_timestamp_load_data_columns())
+        pivoted_cols = set(self.get_pivot_dimension_columns())
+        exclude = time_cols.union(pivoted_cols)
+        dim_cols = [x for x in self._load_data.columns if x not in exclude]
+        df = self._load_data.select(*dim_cols).distinct()
+        return self._remap_dimension_columns(df, mapping_references)
```

Comment:
Since there's no collapsing/aggregation in _remap_dimension_columns, this will contain duplicates if to_id in maps contain duplicates, which seem to go against what the name of the func implies

Reply:
Yeah, this is a bug. It should be sufficient to append a `.distinct()` to the end of this, right?